hbase2/src/main/java/site/ycsb/db/hbase2/HBaseClient2.java

```diff
@@ -165,9 +181,18 @@ public void init() throws DBException {
       debug = true;
     }
 
-    if ("false"
-        .equals(getProperties().getProperty("hbase.usepagefilter", "true"))) {
-      usePageFilter = false;
+    if (isBooleanParamSet("hbase.usepagefilter", true)) {
+      usePageFilter = true;
+    }
```

Comment:
Currently `usePageFilter` defaults to true, so with this implementation there's no way to opt out. replace the if block, e.g.

```
usePageFilter = isBooleanParamSet("hbase.usepagefilter", usePageFilter);
```